apps/web/src/pages/PoolCandidates/ViewPoolCandidatePage/components/MoreActions/MoreActions.tsx

```diff
@@ -0,0 +1,188 @@
+import * as React from "react";
+import { useIntl } from "react-intl";
+import UserCircleIcon from "@heroicons/react/24/outline/UserCircleIcon";
```

Comment:
Can we use the smaller version of this?  Should be from: `@heroicons/react/20/solid/UserCircleIcon";`

This would be the same for the print icon.

Reply:
Fixed in [3706605](https://github.com/GCTC-NTGC/gc-digital-talent/pull/10354/commits/37066056e191ef5229c800ecde1ba667b11207c7)